cl_dll/view.cpp

```diff
@@ -78,7 +78,7 @@ extern cvar_t	*cl_forwardspeed;
 extern cvar_t	*chase_active;
 extern cvar_t	*scr_ofsx, *scr_ofsy, *scr_ofsz;
 extern cvar_t	*cl_vsmoothing;
-extern Vector   dead_viewangles;
+/*extern */ Vector   dead_viewangles;
```

Comment:
Are you sure? Then it's definition should be replaced by declaration in ```input_xash3d.cpp```(e.g. add extern key).

I think same fix(as for input_xash3d.cpp) must be applied for inputw32.cpp.